advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,590 @@
+---
+title: 'Multiple Repositories'
+description: "Deeper look at the multiple repositories support feature"
+---
+
+The multiple repositories support was added in the [v2.33](https://github.com/pgbackrest/pgbackrest/releases/tag/release%2F2.33) release. One example where multiple repositories could be used, would be for situations where a local repository would have minimal retention (for fast restores), and a remote repository would have a longer retention (for redundancy and access across the enterprise).
```

Comment:
The multiple repositories support "feature" was added ....

You can use this feature in a scenario where a local repository would have minimal retention (for fast restore) and a remote repository would have longer retention (for redundancy and enterprise access ).

Reply:
"maximum" retention and "enterprise access" isn't clear enough to me. And I strongly want to emphasis on the fact that it's only 1 example of what this feature is made for.

So, I suggest: _"You can use, e.g., this feature in a scenario where a local repository would have minimal retention (for fast restore) and a remote repository would have longer retention (for redundancy and access across the enterprise)."_

Reply:
new proposal: minimal -> lesser

Reply:
fixed in 8f97564